src/test/java/cloud/eppo/FlagEvaluatorTest.java

```diff
@@ -356,6 +356,94 @@ public void testObfuscated() {
     assertEquals("A", result.getVariation().getValue().stringValue());
   }
 
+  @Test
+  public void testObfuscatedExtraLogging() {
+    // Test that extraLogging is properly deobfuscated when isConfigObfuscated is true
+
+    Map<String, Variation> variations = createVariations("a");
+
+    // Create extraLogging with obfuscated keys and values
+    Map<String, String> obfuscatedExtraLogging = new HashMap<>();
+    obfuscatedExtraLogging.put(base64Encode("testKey"), base64Encode("testValue"));
+    obfuscatedExtraLogging.put(base64Encode("anotherKey"), base64Encode("anotherValue"));
+
+    List<Split> splits = new ArrayList<>();
+    splits.add(new Split("a", null, obfuscatedExtraLogging));
+
+    List<Allocation> allocations = createAllocations("test", splits);
+
+    // Create the base flag
+    FlagConfig flag = createFlag(getMD5Hex("flag"), true, variations, allocations);
+
+    // Encode the variations (following the same pattern as the main obfuscated test)
+    Map<String, Variation> encodedVariations = new HashMap<>();
+    for (Map.Entry<String, Variation> variationEntry : variations.entrySet()) {
+      String encodedVariationKey = base64Encode(variationEntry.getKey());
+      Variation variationToEncode = variationEntry.getValue();
+      Variation newVariation =
+          new Variation(
+              encodedVariationKey,
+              EppoValue.valueOf(base64Encode(variationToEncode.getValue().stringValue())));
+      encodedVariations.put(encodedVariationKey, newVariation);
+    }
+
+    // Encode the allocations
+    List<Allocation> encodedAllocations =
+        allocations.stream()
```

Comment:
We can do this in testing code because we don't need to be Android 21 compatible?